.storybook/preview.tsx

```diff
@@ -1,4 +1,4 @@
-import "@chanzuckerberg/eds-tokens/lib/css/variables.css";
+import "../lib/tokens/css/variables.css";
```

Comment:
doesn't seem great importing from `lib/`, but this may change when we merge in the `next` branch anyways. alt approach is to also output tokens into `.storybook` and `src/` as needed for internal use

Reply:
Seems reasonable to import from lib. Or at least may be better than jumping through hoops.